tests/data_files/rsa_pkcs8_pbes2_pbkdf2_2048_aes128_sha256.pem

```diff
@@ -0,0 +1,30 @@
+-----BEGIN ENCRYPTED PRIVATE KEY-----
```

Comment:
Thanks for mentioning how these were generated in the commit message. However for files in `tests/data_files` we have executable documentation in the form of a `Makefile`. The files are still committed normally, and this Makefile is not part of the build system, but it's the dedicated place to document how the files were generated.

Reply:
Thanks! I didn't know about this Makefile. I have changed the test data to be more consistent with previous data generated, and added some DER tests as well.

Also, I have logged in to this page, but do not have any green arrows or other indication to start the CI.